vue/components/organisms/ripe-image/ripe-image.stories.js

```diff
@@ -17,26 +17,6 @@ storiesOf("Organisms", module)
                 type: Number,
                 default: number("Version", 52)
             },
-            parts: {

```

Comment:
Why?

Reply:
Just to allow testing the changing of model/brand without breaking the sdk, since the parts fed would be from the old one.

However, I just noticed that this might not be needed, I will check.

Reply:
Is this still a problem? I would expect that I could pass (brand, model, parts) then just change (brand, model) and hope that the parts update themselves.